### PR TITLE
Update samples for dataset samples 10.2.4

### PR DESCRIPTION
**Pull request by folmos-at-orange**
---

### TODO Before Asking for a Review
- [x] Rebase your branch to the latest version of `dev` (or `main` for release PRs)
- [x] Make sure all CI workflows are green
- [x] When adding a public feature/fix: Update the `Unreleased` section of `CHANGELOG.md` (no date)
- [x] Self-Review: Review "Files Changed" tab and fix any problems you find
- API Docs (only if there are changes in docstrings, rst files or samples):
  - [x] Check the docs build **without** warning: see the log of the API Docs workflow
  - [x] Check that your changes render well in HTML: download the API Docs artifact and open `index.html`
  - If there are any problems it is faster to iterate by [building locally the API Docs](../blob/dev/doc/README.md#build-the-documentation)

### CODE REVIEW

File: khiops/samples/samples_sklearn.py [hunk 1]
```diff
@@ -215,49 +214,30 @@ def khiops_classifier_multitable_snowflake():
...
+            "Places": (places_df, "AccidentId"),
         },
         "relations": [
             ("Accidents", "Vehicles"),
             ("Vehicles", "Users"),
             ("Accidents", "Places", True),
         ],
     }
-
-    # Load the target variable "Gravity" from the "AccidentsSummary" dataset
```
Comment (popescu-v) [inline khiops/samples/samples_sklearn.py:252]: I would keep the comment, but update it a bit, to "... from the "Accidents" dataset"

File: khiops/samples/samples_sklearn.py [hunk 2]
```diff
@@ -628,49 +606,30 @@ def khiops_encoder_multitable_snowflake():
...
             "Vehicles": (vehicles_df, ["AccidentId", "VehicleId"]),
-            "Users": (users_df.drop("Gravity", axis=1), ["AccidentId", "VehicleId"]),
+            "Users": (users_df, ["AccidentId", "VehicleId"]),
+            "Places": (places_df, "AccidentId"),
         },
         "relations": [
             ("Accidents", "Vehicles"),
-            ("Accidents", "Places", True),
             ("Vehicles", "Users"),
+            ("Accidents", "Places", True),
```
Comment (popescu-v) [inline khiops/samples/samples_sklearn.py:629]: Why was this change needed? The previous list of relations worked just as well, didn't it?

**Review by popescu-v — COMMENTED** [review]
Two comments:
- an improvement suggestion
- a request for qualification.

File: khiops/samples/samples_sklearn.py (hunk 2 above)
Comment (folmos-at-orange) [reply khiops/samples/samples_sklearn.py:629]: Yes, but that way the order is coherent with the other samples and the way tables are loaded.

File: khiops/samples/samples_sklearn.py (hunk 1 above)
Comment (folmos-at-orange) [reply khiops/samples/samples_sklearn.py:252]: Ok

Comment (popescu-v) [reply khiops/samples/samples_sklearn.py:252]: Ok.

File: khiops/samples/samples_sklearn.py (hunk 2 above)
Comment (popescu-v) [reply khiops/samples/samples_sklearn.py:633]: Ok, fair enough.

**Review by popescu-v — APPROVED** [review]
LGTM, but branch HEAD (commit https://github.com/KhiopsML/khiops-python/pull/303/commits/2bc757beea0c7fd2555820bf1f64d5435310e293) should be removed before merging this PR. To do this, the tutorials branch needs to be merged before as well, so that the CI of the current PR is green w/o this commit.

**Comment by folmos-at-orange** [comment]
> LGTM, but branch HEAD (commit [2bc757b](https://github.com/KhiopsML/khiops-python/commit/2bc757beea0c7fd2555820bf1f64d5435310e293)) should be removed before merging this PR. To do this, the tutorials branch needs to be merged before as well, so that the CI of the current PR is green w/o this commit.

- Updated `khiops-python-tutorial`
- Eliminated `TMP` commit
- Merged
